packages/react/src/patterns/sections/CardSection/CardSection.js

```diff
@@ -4,20 +4,17 @@
  * This source code is licensed under the Apache-2.0 license found in the
  * LICENSE file in the root directory of this source tree.
  */
-
 import React, { useEffect, useRef } from 'react';
 import { ArrowRight20 } from '@carbon/icons-react';
 import { Card } from '../../sub-patterns/Card';
 import classNames from 'classnames';
-import { settings as ddsSettings } from '@carbon/ibmdotcom-utilities';
```

Comment:
I see you removed this. I'm wondering if this will need a stable selector.

Reply:
Ok will add back